<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signin</class>
 <widget class="QMainWindow" name="Signin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signin</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Image/Gucci_logo.webp</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>600</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QLabel" name="imag_sigin">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>300</width>
       <height>430</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Image/Gucci_signin.jfif);
border-top-left-radius:25px;
border-bottom-left-radius:25px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_signin_right">
     <property name="geometry">
      <rect>
       <x>293</x>
       <y>30</y>
       <width>300</width>
       <height>430</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(247, 247, 247);
border-top-right-radius:25px;
border-bottom-right-radius:25px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_login">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>60</y>
       <width>271</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gabriola</family>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(0, 0, 0,200);
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LOGIN&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>125</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_userName">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>225</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Semilight</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(195, 188, 191, 0.3);
border-radius:15px;
padding-left:10px;</string>
     </property>
     <property name="maxLength">
      <number>49</number>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>205</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_password">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>230</y>
       <width>225</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(195, 188, 191, 0.3);
border-radius:15px;
padding-left:10px;</string>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_hidden_pw">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>240</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="hidden_pwd" resource="Image.qrc">
       <normaloff>:/Image/hidden.png</normaloff>:/Image/hidden.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login_now">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>305</y>
       <width>130</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_login_now{
	background-color:rgba(157,150,153,0.86);
	border-radius:20px;
}
QPushButton#btn_login_now:hover{
	
	background-color:rgba(102, 94, 95, 0.21);
	border: 3px solid rgba(157,150,153,0.95);
}
QPushButton#btn_login_now:pressed{
	padding-bottom:5px;
	background-color: rgb(117, 117, 117);
	border: 3px solid rgba(157,150,153,0.95);
}
</string>
     </property>
     <property name="text">
      <string>Login Now</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_dont_have_acc">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>375</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Don't have an account? </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_register">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>375</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_minimize_window">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>40</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_minimize_window{
	border:none;
}
QPushButton#btn_minimize_window:hover{
	border:none;
	
	border-radius:12px;
	border-image: url(:/Image/minimize.png);
}
QPushButton#btn_minimize_window:pressed{
	
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_close_window">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>40</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_close_window{
	border:none;
}
QPushButton#btn_close_window:hover{
	border:none;
	border-radius: 12px;
	border-image: url(:/Image/close_window.png);
}
QPushButton#btn_close_window:pressed{
	
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>410</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>125</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Administrator?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_admin">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>5</y>
        <width>70</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QCheckBox:indicator:checked{
	
	image: url(:/Image/switch_on.png);
}
QCheckBox:indicator:unchecked{
	

	image: url(:/Image/switch_off.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lb_notice">
     <property name="geometry">
      <rect>
       <x>355</x>
       <y>280</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
